### DISCUSSION TITLE
Select row after datagrid reload

### DISCUSSION THREAD

**Discussion by martinb69** [Megabit/Blazorise#3905] [Q&A] [answered]
I have a Datagrid components that reads data from a database. 
The user can select a row and edit data in a custom Modal form.
When the edit is complete I reload the data from the database but I am not sure how to retain the current selection.
I have tried the @bind-SelectedRow but this does not select the previous selected row after the reload.
Any tips on how I can update the grid and keep the selected row?
Thanks!

**Comment by David-Moreira** [comment, marked as answer]
Make sure to keep track of the SelectedRow. 

Make use of the `Select` API after the data has been reloaded.
![image](https://user-images.githubusercontent.com/22283161/174903041-878afce5-fc90-4a6c-b301-631d32fd0c98.png)
(We do need to make the public exposed methods avaialble on our docs)

Let us know if this works & helps you.

**Reply by martinb69** [reply]
I overlooked this one and got it working. Thanks!
